search/scorer/scorer_term.go

```diff
@@ -114,6 +132,65 @@ func (s *TermQueryScorer) SetQueryNorm(qnorm float64) {
 	}
 }
 
+// multiplies deciding how much does a doc length affect the score and also
+// how much can the term frequency affect the score in BM25 scoring
+var k1 float64 = 1.2
+var b float64 = 0.75
```

Comment:
Like we discussed, no point in making these `var`s and keeping them unexported.
Perhaps export them and prefix a `BM25` to indicate what they're applicable to.